docs/guides/Checkout/orders-2/create-a-regular-order-from-an-existing-cart.md

```diff
@@ -0,0 +1,222 @@
+---
+title: "Create a regular order from an existing cart"
+slug: "create-a-regular-order-from-an-existing-cart"
+hidden: false
+createdAt: ""
+updatedAt: ""
+---
+The shopping cart is where the information of the products chosen by the customer while browsing a store is gathered. This data may include item prices, shipping value, payment, and delivery methods, among others.
+
+This guide will describe how to create an order from an existing shopping cart by the API.
+
+> ℹ️ To place an order from a new cart, access the [Create a regular order using the Checkout API](https://developers.vtex.com/docs/guides/create-a-regular-order-using-the-checkout-api).
+
+## Getting shopping cart information
+
+The first step is to get the `orderFormId` of the shopping cart you want to create an order. For more information, access the [Get cart information by ID](https://developers.vtex.com/vtex-rest-api/docs/get-cart-information-by-id) guide.
+
+## Placing order from an existing cart
+
+To create an order via a specific shopping cart, you need to use the [Place order from an existing cart](https://developers.vtex.com/docs/api-reference/checkout-api#post-/api/checkout/pub/orderForm/-orderFormId-/transaction) endpoint. In this request, you must send the `orderFormId` through the URL address, as shown by the example below:
+
+`https://{accountName}.{environment.com.br}/api/checkout/pub/orderForm/ede846222cd44046ba6c638442c3505a/transaction`
+
+Additionally, you need to send the request body containing the following information:
+
+- `referenceId`: ID of the orderForm corresponding to the cart in which the order will be created (`orderFormId`).
+- `savePersonalData`: this parameter should be set as "true" if the shopper’s data provided during checkout should be saved for future reference. Default value is "false".
+- `optinNewsLetter`: this parameter should be set as "true" if the shopper opted to receive the newsletter. Default value is "false".
+- `value`: Total value of the order without separating cents. For example, $24.99 is represented by `2499`.
+- `referenceValue`: reference value of the order for calculating interest (if applicable). Can be equal to the total value and does not separate cents. For example, $24.99 is represented by `2499`.
```

Comment:
```suggestion
- `referenceValue`: reference value of the order for calculating interest (if applicable). It can be equal to the total value and does not separate cents. For example, $24.99 is represented by `2499`.
```